docs/smart-contracts/tokens/wayfinding.mdx

```diff
@@ -0,0 +1,80 @@
+
+## Issuing assets on Stellar
+
+Stellar has first-class support for asset tokenization — issuing an asset can be done using a [built-in transaction](https://developers.stellar.org/docs) without the development of a smart contract.
+
+Stellar’s transactions are fast and cost-effective, making the network great for remittances and micropayments. It also has built-in features for compliance, asset management, and auditing. If you are looking to perform transfers of value, issuing assets on Stellar has all the needed capabilities.
+
+Stellar assets:
+
+- Are compatible with Stellar ecosystem products (such as Stellar wallets) and other ecosystem products (such as exchanges).
+- Benefit from [anchors](../../learn/encyclopedia/anchors.mdx), the bridges between the Stellar network and traditional financial systems. Explore the global [Stellar anchor directory](https://anchors.stellar.org/) for further details.
+- Give the issuer granular control over asset management with features that allow the issuer to [name the asset](../../issuing-assets/control-asset-access.mdx#naming-an-asset), [determine access control](../../issuing-assets/control-asset-access.mdx#controlling-access-to-an-asset-with-flags), [limit asset supply](../../issuing-assets/control-asset-access.mdx#limiting-the-supply-of-an-asset), [publish asset information](../../issuing-assets/publishing-asset-info.mdx), and [ensure compliance](../../issuing-assets/anatomy-of-an-asset.mdx#compliance).
+
+:::note
+
+Note that while these items are also possible with custom smart contract tokens, it is more work to build the token contract rather than using the already-implemented features of Stellar asset tokens.
+
+:::
+
+Assets issued on the Stellar network are accessible to smart contracts with the use of that asset’s Stellar Asset Contract (SAC).
+
+### Stellar Asset Contract
+
+The Stellar Asset Contract (SAC) is compiled into the protocol layer and allows smart contracts to interact with assets issued on Stellar. A single instance of the SAC can be deployed for every Stellar asset by anyone who wants to interact with the asset from a contract. The SAC has access to all account (for XLM) and trustline (for all other assets) balances as well as smart contract token balances.
+
+Read more about the SAC [here](./stellar-asset-contract.mdx).
+
+Learn how to deploy a Stellar Asset Contract for an asset in [this How-To Guide](../../smart-contracts/guides/cli/deploy-stellar-asset-contract.mdx).
+
+**Benefits of the SAC:**
+
+- Compatibility: the SAC benefits from Stellar assets' existing interoperability.
+- Cost and resource efficiency: the SAC is built into the protocol-level code that already runs the network. So, there is no need to spin up a Wasm contract’s VM to invoke a function. Each function within the SAC will be more resource-efficient than its custom-coded counterpart.
```

Comment:
Abbreviated to remove some terms that don't show up often in dev vocabularies. e.g. "protocol level code."
```suggestion
- Cost and resource efficiency: the SAC is built into the protocol instead of being a contract that runs in a virtual machine. Each function within the SAC will be more resource-efficient than its custom-coded counterpart.
```